hostscripts/clouddata/syncSLErepos

```diff
@@ -257,13 +257,7 @@ for version in 5 4 2.1; do
 
         echo ================== Updating Storage $version
 
-        # sync version 5 from ibs while in development
-        poolsource=
-        if [[ $version == 5 ]]; then
```

Comment:
maybe just change this to 6? that effectly does the same but keeps the code path around once we enable mirroring of SES6